### PR TITLE
Updates to fix final misc spacing issues and cleanup a few things.

### LINKED ISSUE #1514: static content - misc spacing
- [x] lets have all content start a little lower than it is (see old people page and how that looks with no header)
- [x] ~~50px margin at the bottom of all of the page's content wrapper~~ 
- [x] spacing (leading) between lines is a bit awkward sometimes (like job/internship listings), Crystal + Barry discussed this as well
- [x] eliminate gap between intro and additional text when there isn’t a gallery
- [x] add a bit more top margin to CTA's on static pages.

### PR DESCRIPTION
**Pull request by weerd**
Misc spacing edits for the win!

Resolves #1514

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     <?php if (isset($intro)): ?>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:13]: Shouldn't this be a check for `$intro_title` and not `$intro` ?

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:13]: That was already there. There are scenarios (old people page) where the title is absent but intro has content.

Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:13]: oh, okie

**Comment by DFurnes** [comment]
Cool code :snowboarder:

**Pull request merged**